templates/components/loaders/python/loader.py

```diff
@@ -0,0 +1,13 @@
+import os
+import importlib
+
+
+def get_documents():
+    # For each file in .loaders, import the module and call the get_documents function
+    documents = []
+    for loader in os.listdir(os.path.join(os.path.dirname(__file__), "loaders")):
+        if loader.endswith(".py"):
+            loader = loader[:-3]
+            module = importlib.import_module(f"app.engine.loaders.{loader}")
```

Comment:
1. so we just import one file per loader? this means the loader files must have different names - we should probably write a comment
2. do we copy this loader only if we have multiple datasources?

Reply:
1. Yes, we only copy one file per loader. 
There is another approach which is more straightforward to the users is specifying all the load_documents from each loader in a __init__.py file in the loaders folder, but this approach requires generate init file dynamically so i did not implement that way.
2. Yes, we still copy this loader if there is only one data source.

Reply:
if we always copy this loader, how about we always copy the file (or llamaparse) and the web loader too?
we could then also make this import not dynamic but explicit